test/built-ins/Temporal/PlainDateTime/prototype/round/round-toward-big-bang.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
I think this is redundant with `test/built-ins/Temporal/PlainDateTime/prototype/round/rounding-direction.js` in #3468

Reply:
Yeah -- I deleted it. I didn't know about Philip's work there. Good catch!